src/llvm-multiversioning.cpp

```diff
@@ -697,18 +698,59 @@ Constant *CloneCtx::rewrite_gv_init(const Stack& stack)
     return res;
 }
 
+// replace an alias to a function with a trampoline and (uninitialized) global variable slot
+std::pair<Function *, GlobalVariable *> CloneCtx::rewrite_alias(GlobalAlias *alias)
+{
+    Function *F = cast<Function>(alias->getAliasee());
```

Comment:
And we need to check to make sure the calling convension is unchanged. For now an `assert(!is_vector(F->getFunctionType()))` should be enough.